supernode/daemon/mgr/cdn/manager.go

```diff
@@ -103,18 +104,16 @@ func (cm *Manager) TriggerCDN(ctx context.Context, task *types.TaskInfo) (*types
 
 	return &types.TaskInfo{
 		CdnStatus:  types.TaskInfoCdnStatusSUCCESS,
-		FileLength: realFileLength,
+		FileLength: downloadMetadata.realFileLength,
 		RealMd5:    realMD5,
 	}, nil
 }
 
 // GetHTTPPath returns the http download path of taskID.
+// The returned path joined the DownloadRaw.Bucket and DownloadRaw.Key.
 func (cm *Manager) GetHTTPPath(ctx context.Context, taskID string) (string, error) {
```

Comment:
@lowzj I have updated the implementation of GetHTTPPath. Dfget will get resources from supernode instead of get them form storage directly. So the supernode should do these things:

return a path that can get info to download resources from storage
listen the download port and accept the download requests
parse the path and get resources from storage.

So, I think it's reasonable to do this. WDYT?